### DISCUSSION TITLE
<details> content rendered outside

### DISCUSSION THREAD

**Discussion by GuillaumeDesforges** [mkdocs/mkdocs#3180] [Q&A]
I'm using mkdocs with markdown akin to that:

```markdown
  * a bullet point, renders well

    > A quote, renders well

    <details>
      <summary>A summary, renders well</summary>

      Some details, excuse me?
    </details>
```

renders

```html
<ul>
<li>
<p>a bullet point, renders well</p>
<blockquote>
<p>A quote, renders well</p>
</blockquote>
<p></p>
<details>
<summary>A summary, renders well</summary>
</details>
<p></p>
<p>Some details, excuse me?</p>
</li>
</ul>
```

given configuration

```yaml
nav:
  - ...

plugins:
  - same-dir
```

As we can see here, the content inside the `<details>` block in the source gets put outside in the output HTML.

The expected behavior for me would be that the closing `</details>` would be after the HTML of its content.

**Comment by oprypin** [comment]
The Markdown implementation is fully defined by https://github.com/Python-Markdown/markdown, so not solvable in this repository.

However instead of using HTML you could switch to a Markdown-native syntax for a details tag, provided by an extension. Some options are:

* https://oprypin.github.io/markdown-callouts/#collapsible-block-syntax
* https://facelessuser.github.io/pymdown-extensions/extensions/details/

**Reply by GuillaumeDesforges** [reply]
Thanks for the pointers. I've re-created the issue on the markdown implementation repo. https://github.com/Python-Markdown/markdown/issues/1333
